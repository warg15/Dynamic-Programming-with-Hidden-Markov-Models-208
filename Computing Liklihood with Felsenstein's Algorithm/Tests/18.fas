>Emperor-penguin
GGGTCTAAAATACAAGGCGGGTGTGATCAAAGCTTCATATGTTAAGGGTGGTCGTTACTACATGAGAACGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGATTATGGACGTCTCTGCAAAAGTTGCATGCGGCTGCGGAGCCCGTATAGCTCTGCGGTAAGGAGCGTAGGTACTTCTGAATTGGAGGTTCCTGAC
>Rhinoceros-hornbill
GTGTCTAAAATACAAGGCGAGTGTGATCGAAGCTTTATTTGCTAAGGATGGTCATTACTACATGAAAACGATTGTACGGCAAGTGCAACGGTTATAGATTGCAGAGTTATGGACGTCTGTGCAAAAGTTGCACGCGGCTGGGGGGTACGTATAGCACTGCGGAAGGGAGCGTACGTACTTTCGAATTGGAGGTTCTCGAC
>Cuckoo-roller
GGGTCTAAAATACAAGGCGGGTGTGATCAAAGCTTCATATGCTGAGGATGGTACTTACTACATGAAAACGATCCTACGGCAGGTACAACGGTTATAGATTGCAGGGTTGTGGACGTCTCTGCAAAAGTGGCATGCGGCTGCGGAGCCCCTATCGCTCTGCGATGAGGAACGTAGGTGCTTTTGAATTGGAGGTTCTTGAC
>Annas-hummingbird
GGGTTTAAAATACGAGGCGGGGGCGATCAAAGCTTCCTATGTTAAGGATGGTTGTTACTACATGAAGCCAATCGTACGCCAGGTACAATGATTCTAGATTGGAGAGTTATGGACCTCTATGCAAAAGTTGGATGAGGCTGCGGAGCCCGTATAACTCTGCAGTGGGGAACGTAGGTGCTTCTAAAATGGAGGTTCTTGAC
>Medium-ground-finch
GGGTCTAAAATAGAGGGCGGGTGTGATCGAAGCTTCATACACCAAGGTTGGTTGTCACTACATGAAAACGATCGTACGGCAGGTACAAAGATTATAGATTGGAGGGGTATGGACGTTTCTGCAAAAGTTGCGTGCGAGTGCAGAGCTCGTATAGCTCTGCGGTGGGGGATGGAGGTATTTTTGATTAGGAGGTTCTTGAC
>American-crow
GGGTCTAAAATAGAAGGCGTGTGTGATCGAAGCTTCATACGCTAAGGATGGGTGTCACTACATGAAAACGATCGTACGACAGGTACAACGATTATCGTTTGGAGGGTTATGGACGGTTCTGCAAAAGTTGCGTACGAGTGTAGAGCTCGTATAGCTCTGCGGTGGGGAATGGAGGTATTTTTGAATAAGAGGTTCTTGAC
>Zebra-finch
GGGCCTAAACTAGAAGGCGGGTGTGATCGAAGCTTCATACGCTAAGGATGGTTGTCAATACATGAAAAAGATAGCACGGCAAGTACAACGATTATAGAGTGGAGGGGTATGGACGTTTCTGCAAATGTTGCGTGCGAGTGCAGAGCTCGTATAGTTCTGCGGTGGGGGATGGAGGTATTCTTGATTAGGAGGTTCTTGGC
>White-throated-tinamou
GGTTCCAAAATACAAGGCGGATAAGATCAAAATTTCATATGTTAAGGGTGGTTGTCACCACACGAAAATAGCCGTGCAGCAGGAACAACGGGTATGGATTGGGGGGTTATAGACGTGTTTGTGAAAGTTGCATATGGCTGCGGACCGCGTATAGCTCTGTGGTGGGGAACGTAGAGACTTCTCGATTGTTGATTATTTAC
>Crested-ibis
GGATCTAAAATACAAGACGGGTGTGATCAAAGCTTCATATGTTAAGGATGGTTGTTACTACATGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGGTTATGGACGTCTCTGCAAAAGTTGCATACGGCTGCGGAGCCCGTATAGCTCTGCGGTCAGGAACGTAAGTACTTCTGAATTGGAGGTTCTTGAC
>Carmine-bee-eater
GGGTTTAAAATATAAGGCGGGTGTAAACAAACCTTCATATGCTCAGCATGGTTCTTACAACATGAAAACAATCGCACGGCATGTACAACGGTCAAAGATTGGGGGGTTACGGACGTCTGTACAGAAGCTGCATGCGGCTGCGGAGTCCGTATAGCTCTGCGGTATGGAACGGAGGTACCTTTGAATTGGAGGTTCTTATC
>Red-crested-turaco
GGGTATAACACACAAGGCGGGTGTGATCAAAGCTTCATATGTTAAGAATGGGTGTTACTACATGAAAACGATCGTACCGCAGGTACAACGGTTATAGATTGGAGAGTTATGGACGTCTCTGTAAAAGTTGCATGCGGCTGCGGAGCCCATATAGCTCTGCGGTGGGGAACGTAGGTACGTCTGAATTGGGGGTTCTTGAT
>Adelie-penguin
GGGTCTAAAATACAAGGCGGGTGTGATCAAAGCTTCATATGTTAAGGATGGTTGTTACTACATGAGAAGGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGATTATGGACGTCTCTGCAAAAGTTGCATGCGGCTGCGGAGCCCGTATAGCTCTGCGGTGAGGAGCGTAGGTACTTCTGAATTGGAGGTTCTTGAC
>Chicken
GGATCTAAAATACCAGGTGGGTGTGCTTAAATCCTCAAATGTTAAGGATGGTTGTTACTATATGAAAACGTCGGCCCGGCAGAAAAAACGGTTGTAGATTGGAGGGTTAGATACGTCTCCTCGAAAGTTTCATGCGGCTGCGGAACGCCGATAGGTTGACGGCGGGAATCGTAGGTACTTCTGACTTGGAGGTTCTCGAC
>Red-legged-seriema
GGGTCTAAAATGCAAGGCGGGTGTGATCAAAGCTTCATATGTTAAGAATGGTTGTTACGACATGAATACGATCGTACGGCGGGTACAACGGTTATAGATTGGAGGGTTATGGAGGTCTCTGCAAAAGTTGTGTGCGGATGCGGAGCCCGTATAGCTCTGCGGTGGGGAATGTAGGTACTTTTGAATTGGAGGTTCTTAAC
>Yellow-throated-sandgrouse
GGGTCTAAGATACAAGGCGAGTGTGATCAAAGCTTCATATGTTGAGGATGATTGTTACTACATGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTCGAGGGTTATGGACGTCTCAGCGAAAATTGCATCCAGCTGTGGACCCCGTATAGCTCTGCGGTGACGAACGTAGGTACTTCTGAAGTGGAGGTTCTTGAC
>Pigeon
GGGTCTAAAATACAAGGCAGGTGTGAGCAAGGCGTTATATGTTAAGGATAGTTGTTACTGCATGAAAGCGATCGTACGGCAGGTACGACGGTTATAGATTGGAGGGTGGTGGACGTCTCTGCAAAGGTTGCATCCGGCTGCGAAGCCCGCATAGCTCTGCGGCGGGGAACCTAGGTACTTCTGAATTGGAGGTTCTTAAC
>Budgerigar
AGGTCTAATATAAAGGGCGGGTGTGATCAAAGCTCCATATGTTAAGGATGGTTGTTACTACATGAAAACGACCGTGCAGCAGGTAAAATGGTTAGATATTGGAGGTTTATGGACGTCTCTGCATAAGTTGCGTGCGGGTCCAGAGCCCATACAGCTCTACAGTGGGGAACGTAGGTCCTTTTAGAGTCTAGGTTCTTGAC
>Peregrine-falcon
GAGCCTAAGATACATGGCGAGTGTGATCAAAGCTTCATATGTTAAGGATGGTTGTTACTGCATGAAAACGATCGTACGGCAGGTACAACGGTTACAGATTGGAGGGTTATGGACGCCTTTGCAAAGGTTGCGTACGGGTGCAGAGCCCGCATGGGTCTGCGATGGGGAACGTAGGTACTTTTGAATTGGAGGTTCTTGAC
>Sunbittern
GGGTTTAAAAGAAAAGGCGGGTGTGATCAAAGATTCATATGTTAAGGACGGTTGTTACTACATCAAAACGGTCGTATGGCCGGTACACCGGTTGAATATTGGAGAGTTATGAACGTCTCTGCAAAAGTTGCATGCGACTACGGAACCAGTATAGCTCTGCAGTGAGGAACGTAGGCACTTCAGAATCTGAGGTTTTTGAC
>Golden-collared-manakin
GGGGCTAAAATACAAGGCGGGTGTGATCAAAGCTTCACATACTAAGGATGGTTGTTACTACATGAAAACGGTCGTAGGGCAGGTACAACGATTATAGATTGGAGGGTTATGGACGTTTCTGCAAAAGTTACTTGCGAGTGCAGAGCTCGTGTAGCTCTGCGGTGGGGGATCTAGGTATTTTTGAATTGGAGGTTCTCGAC
>Speckled-mousebird
GGGTCTAAAATACATGGCGGGTGTAGTCAAAACATTATATATTACGGATGATTGTGATGATATGAAAACGATAGTACCGTAGGTACGACGGTAATAGGTTGCAGGGTTATGGACGTCTCTGCAAGAGTGGCATGCGGCTGCGGAGCCCGTATAGCTCTGCGGTGGGGAACGTAGGTACTTTCGAATTGGAGGTTCTTGAC
>Northern-fulmar
GGGTCTAAAATACAAGGCGGGTGTGATCAAAGCTTCATATGTTAAGGATGGTTGTTACCACATGAAGACGATCGTACTGCAGGTACAACGGTTATAGATTGGAGGGTTATGGACGTATCTGCAAAAGTTGCATGCGGCTGCGGAGCCCGTATAACTCTGCGGTGAGGAACGTAGGTGCTTCTGAATTGGAGGTTCTTAAC
>Common-cuckoo
GGGCCTAAAATACAAGACGGGTGTGATCGAAGCCTCATATGTTGAGGATGGTTGTTACTACATGAAAATGATCGTACGGACGGTACAACGGTTGTAGATTGGAAGGCTATGGACGTCTCTGCAAAGGTTGCACGCAGCTGCGGAGCCCGTATAGCTCTGCAGTGGGGAACGTGAATGCTACTGAATTGAAGGTTCTTGAC
>Grey-crowned-crane
GAGTCTAAAATACAAGGCGGGTGTGATCAAAGCTTCATATGTTAAGGACGATTGTTACTATACGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGGTTATGGACGTCTCTGCAAAACTGGCATGCGGCTGCGGAGCCCGTATTGCTCTGCGGTGGGGAACGTAGGTACTTCTGAATTGGAGGTTCTTGAC
>Brown-mesite
GGGTCCAAAATACAAGGCGGGTGTAATCAAAGCTTCATATGTTGAGGATGGTCGTTACCACATGAAAACGATCGTACGGCAGGTATAACGGTTATAGATTGGAGGATTATGGACGTCTCTGCAAAAATTGCATCCGGCTGCGGAGCCGGTACAGCCCTGCGGTGCGGAACGCAGATACTTCTGAATTGGCGGTTCGAGAC
>Red-throated-loon
GGGTCTAAAATACAACGCGGGTATGACCAAAGCTTCATATGCTAAGGATGGTTGGTACTACATGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGGTTATGGTCGTCTCTGCAAAAGTTGCAAGCGGCTGCGGAGCCCGTATAGCTCTGCGATGAGGAACGTAGGTACTTATGAGTTGGGGGTTCTTCAC
>Bar-tailed-trogon
GGGTCTAAAATATAATGCGGGTGTGATCAGCGCTTCATATGTTAAAGATGGTTCTTACTACATGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGGTGATGGACGTCTTTGCAAAAGTTACATGCGGCTGCGGAGCCCAGATAGTTCTGCGGTAGGGAACGTAGGTACTTTTGAATTGGAGGTTATTGAC
>Bald-eagle
GGGTCTAAAATACAAAGCGGGTGTGATCAAAGCTTCATATGCTAAGGATGATTCTTACTACATGAAAACGATCGTACGGCAAGTACAACGGTTATAGATTGGAGCGTTATGGACGTCTCTGCAAAAGCTGCATGCGGCTGCAGAGCCCGTATAGCTCTGCGGTGGAGAACGTCGGTACTTTTGAATTGAAGGTTCTTGAC
>Chimney-swift
GGGTCTAAAATACAAGGCGGGCGTGATCAGAGCTTCATATGTTAAGGACGGTTGTTACTACATGAAGACAATCGTGCGGCAGGTACAACGGTTATAAATTGTAGGGTTATGGACGTCTCTGCAAGGGTTGCATGCGGCTACGGGGCCCGTATAGCTCTGAGGTGGGGAACGTATGTACTTCTGAATTGGAGGTCCCTGAC
>White-tailed-eagle
GGGTCTAAAATACAAAGCGGGTGTGATCAAAGCTTCATATGCTAAGGATGATTCTTACTACATGAACACGATCGTACGGCAAGTACAACGGTTATAGATTGGAGCGTTATGGACGTCTCTGCAAAAGCTGCATGCGGCTGCAGAGCCCGTATAGCTCTGCGGTGGAGAACGTCGGTACTTTTGAATTGAAGGTTCTTGAC
>Hoatzin
GGGTCTGAAATACAAGGCGGGTGTGACCAAAGCCTCATATTTTAAGGATGGTTGTAACTGCACGAAAACGATCGGACAGCAGGTACAACGGTTATGGATTGGAAGGTTATGGACGTCTCTGTAAAAGTTGCATGCAGCGGCGGAGCCCGTGCAGCTCTACGGCGGGGAACGTAGATACTTCTGAATTGGAGGCTCTTGAC
>Barn-owl
GGGTCTAAAATACAAGGCGGGAGTGATCAAAGCTTCATATGCTAAGGATGGTTGTTACTACGTAAAAATGATCGTACGGCAGGTACAACCGTTATAGACTGGAGGGTTATGGACGTCTTTGCAAAAGTTGTACGCAGTTGCGGAGCCCGTATAGCTCTGCGGTGGGGAACGTAGGTACATTTGAATGGGAGGTTCTTGAC
>Killdeer
GAGTCTAAAATACAAGGCGGGTGTGATCAAAGCTTCACATGTTAAGGATGATTGTTACTACATGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTGTAGGGTTATGGACGTCTCCGCAAAAGTTGCATGCGGTTGCGGAGCCCGTATAGCTCTGCGGTGGGGAACGTAGGTACTTCTGAATCGGAGGTTCTTGAC
>Chuck-wills-widow
GGGTCTAAGATACAAGGCGGGGGTGATCAAAGCTTCAAATGTTAAAGATGGTTGTTACTACATAAAAACGATCGTGCGGCAGGGACAACGGCTATACATTGGAGGGTTGTGGACGTCTCTGCAAAAGTTTCATGCGACTGCGGAGCCCGTATAGCTCTGCGGTGGGGAAAGTAGGTACTTCTGAATTGGAGGTTCTTGAC
>Pekin-duck
GGATCTAAAATACCAGGCGGGTGTGATCAAAGCATCATATCTTAGAGATGGTTGTTACTACACGAAAACGACCGTCCGTCAGATGCAACGGTTGTAGATTCGAGGATTAGGGCTGTCTCTACAAATGTCGGATGCGGCTGCGGAGCCCAGATAGGTTTACGGTGCGGAACGTAGGTACTTCTGAATTGGAGGTTCTTAAC
>Great-crested-grebe
GGGTCTAAAACACAAGGCGGGTGTGATCAAAGCTGCCTATGTTAAAGATTGCTGTTACTACATGAAAACTATCGTACGGCAGGTACAACCGTTGTAGATTGGAGGGTTATGGACGTCTCTGCAAAAGTTGCATGCGGCTGCGGAGCCGATATAGCTCTGCGGTGGGGAACGTAGGTACTATTGAATTGGACGTTCTTGAC
>Downy-woodpecker
TGGTTTAAAATACAAGGCGGGTATAATCAATGCTTCATATGTTAAGCATGGTTTTGACTACATGAAAACAAGCGTACGGCAGGGACAGGGGTTATAAATTGGAAGGTTATGAACGTCTCTGCAAAAGTTGTGTGCGGTTACGGAGCGCGTATAGCTGTGCGGTGGGGAATGGAGGTACTTTTTAATTGGAGGTTCTTGAC
>Turkey-vulture
GGGTCTAAAATACAAGGCGGGTGCGATCAAAGCTTCATATGCTAAGGATGGTTGTTACTACATGAAAACGATCGTATGGCAGGTACAACGGTTACAGATTGGAGGGTTATGGACGTCTCTGCGAAAGTTGCATGCGGCTACGGAGCCCGTATAGCTCTGCGGTGGGGAACGTAGGTACTTTTGAATTGGCGGTTCTTGAC
>White-tailed-tropicbird
GGGTCTAAAATACAAGGCGGGTTTGATCAAAGCTTCATATGTTAAGGACGGTTGTTACTACATGAAAACGATCGTGCGGCAGCTACAACGGTTGTGGCTTGGAGGGTTATGGACGTCTCTGCAAAAGTTGCATGCGGTTGCGGAGCCCTTACAGCCCTGCGGTGAGGAACGTAGGTACTTCAGAATTGGAGGTTCTTGAC
>Dalmatian-pelican
GGGTCTAAAATACAAGGCGGGTGTGATCAAAGATTCATATGTTAAGGATGGTTGTTACTATATGGAAACGATCGTGCGGCAGGTGCGACGGTTCTAGATTGGAGGGTTGTGGACGTCACTGCAAAAGTTGCATGCGGCGGCTGAGCCCGTATAGTTCTGCAGTGAGGAACGTAGGTACTTCTGAATTGGATGTTCTTGAC
>Rifleman
GGGTCTAAAATACAAGGAGGGTGTAATCGAAGCTTCACATGCTAAGGATGGTCGTTATTACATAAAAATGGTCGTACGGCAAGTACAACAATTATAGATTGGAGGGTTATGGATGTCTCGGCAAAAGTTGCGTGCGAGTGCAGAGCTCGTGTAGATCTGCGGTGGGGGATGTAGGTATTTTTGAATTGGAGGTTCTTGAC
>Kea
GGGACTAATGTAAAGGTCGGGTGTGATCAAAGCTCCATATGTTAGAAATGGTTGTTACTACATGAAAACGACCGTACAGCAGGTATAACGGTTATAGATTGGAGGGTTATGCATGTCTCCGCAAAAGTTGCGTGCGGATCCAGAGGCCATATAGCTCTACGGTGGAGAACGTAGGTACTTTTAAACTGTAGGTTCTTGAC
>Turkey
GGATCTAAAATACCAGGCGGGTGTACTTAAATCCTCAAATGTTAAGGATGGTTGTTACTATATGAAAACGTCGGCCCGGCAGAAACAACGGTTGTAGATTGGAGGGTTAGATACGTCTCCTCGAAAGTTATATGCGGCTGCGGAACCCCGATAGGTTTACGGTGGGAAACGTAGGTACTTCTGACTTGGAGGTTCTCGAC
>Little-egret
GGGTCTAGAATACAAGGTGGGTGTTATCATAGCTTCATATGTTAAGGATGGTTGTTAGTACATGAAAACGATCGTACGGCAGGTACAACGGTTATAGATTGGAGGGTTATGGACGTCTCTGCAAAAGTTGCATGCAGCTGCGGAGCCCGTATAGCTCTGCGGTGAGGGACGTAGGTACTTCTGAATTGGAGGTTCTTGAC
>Common-ostrich
GGTTCCAAAATACAAGGCGGATGTGATCAAAGTTTCATATGTTAAGGATGGTTGGTACCACATGAAAACAACCGTCCAGCCTGGACAACGGGTATCGATTGGGGGGTTATAGACGTCTTTGTAAAAGTTGCATACGGCTGCGAAGCGCGTATAGAACTGTGGCGAGGAACGTAGGTACTTCCCGATTGTATGTTACTTAC
>Great-cormorant
GGGTCTAAAATACAAGGCAGGTTTGATCAAAGCTTCATATGTTAAGGATGGTTGTTACTACATGAAAACGATCGTACGGCGGGTACAACGGTTATAAATTGGAGGGTTATAGACGCCTCTGCAAAAGTTGCATGCGGCTGCGGAGCCGGTATAGCTCTGCGGTGATGAACGTAGGTACTTCTGAATTGGAGGTTCTTGAC
>MacQueens-bustard
AGGTCTAAAATACAAGGCGGGTGTATTCAAAGCATCATATGTTAAGGATAGTTGTTACTACATGAAAACGATCGTACAGCAGGTACAACGGTTATAGATTGGAGGGGTATGGACGTCTCTGCAAAAGTTGCATGCGACTGCGGAGCCCGTATAGCTCTGCGGTGGGGAACGTAAGTACTTCTGAATTGGGGGTTCTTGAC
>American-flamingo
GGGTTTAAAATACAAGGCGGGTGTGATCAAAGATTCATATGTTAAGGATGGTTGTTACTATATGAAAACGATCGTACGGCAGGTACAACGGTTATAGAGTGGAGGGTTATGGACGTCTCTTCAAAAGTTGCATGCGACTGCGGAGCCCGTATACCTCTGCGGTGGGGAACGTAGGTACTTCTGAATTGGAGGTTCTTGAC
