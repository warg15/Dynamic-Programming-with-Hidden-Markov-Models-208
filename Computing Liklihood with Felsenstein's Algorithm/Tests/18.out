-3487.7559631563054
